test/utils/image/manifest.go

```diff
@@ -69,6 +70,7 @@ func initReg() RegistryList {
 		DockerLibraryRegistry:   "docker.io/library",
 		DockerGluster:           "docker.io/gluster",
 		E2eRegistry:             "gcr.io/kubernetes-e2e-test-images",
+		PromoterE2eRegistry:     "us.gcr.io/k8s-artifacts-prod/e2e-test-images",
```

Comment:
Did I miss something, why is this moving to `us` only?

Reply:
Well, it's just a registry I've picked from here: https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/manifests/k8s-staging-e2e-test-images/promoter-manifest.yaml

Reply:
I think you should just do `gcr.io`, I believe it should pick the closest one to where the tests are running when the image is actually pulled.

Reply:
There is no ``gcr.io/k8s-artifacts-prod/e2e-test-images``, so I can't do that. That would have been my first choice too.

Reply:
Hmm. Ok. Need to understand this better.

Reply:
After the [vanity domain flip](https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/Vanity-Domain-Flip.md), this can be just `k8s.gcr.io/k8s-artifacts-prod/e2e-test-images`. Until then I think this is fine as-is.

Reply:
Awesome, thanks @listx 

@claudiubelu since the 2.10 change merged, please update to 2.10 then I think we are good for this one. Well, I am anyway...

Reply:
We should add a TODO or a tracking issue to switch to k8s.gcr.io after the vanity domain flip